community/index/src/main/java/org/neo4j/index/gbptree/GenSafePointer.java

```diff
@@ -47,6 +47,8 @@
     // unsigned int
```

Comment:
Can we rename this class to GenerationalPointer. GenSafe bring a feeling of char greediness when readability is the only victim.